nengo_loihi/snips/nengo_io.c.template

```diff
@@ -58,11 +58,15 @@ void nengo_io(runState *s) {
     // printf("count %d\n", count[0]);
 
     // --- read input values (once every IO_STEPS)
-    if (s->time % IO_STEPS == 0) {
+    // Note that we do this at the *start* of an IO_STEP period (since
+    //  s->time starts at 1), so that we don't have a period at the beginning
+    //  of the simulation where we are ignoring the input.
+    if (s->time % IO_STEPS == 1 % IO_STEPS) {
```

Comment:
Is this the same as `(IO_STEPS==1) || (s->time % IO_STEPS == 1)`? Currently we do it one way here and another way down below. I'd keep it consistent. Also, as elegant as this way is, I think I prefer the explicit OR.